syncer/syncer.go

```diff
@@ -342,21 +345,29 @@ func (s *Syncer) Init(ctx context.Context) (err error) {
 		return err
 	}
 
+	var schemaMap map[string]string
+	var tableMap map[string]map[string]string
+	if s.SourceTableNamesFlavor == utils.LCTableNamesSensitive {
+		allTables, err1 := utils.FetchAllDoTables(ctx, s.fromDB.BaseDB.DB, s.baList)
```

Comment:
maybe add a TODO to reuse the result of allTables, there're `s.fromDB.fetchAllDoTables(ctx, s.baList)` called elsewhere